src/app/core/site-header/site-header.component.html

```diff
@@ -6,20 +6,100 @@
           <span class="visually-hidden">{{ "MAIN_PAGE" | translate }}</span>
           <img class="navbar-brand herb" [src]="'herb.png' | path: 'imagesPath'" alt="herb" />
         </a>
-        <a href="/" class="navbar-brand">
-          <span class="visually-hidden">{{ "MAIN_PAGE" | translate }}</span>
-          <img
-            class="logo"
-            [src]="'kprm.svg' | path: 'imagesPath'"
-            [alt]="'CHANCELLERY_OF_PRIME_MINISTER' | translate"
-          />
-        </a>
-        <a href="/" class="navbar-brand">
-          <span class="visually-hidden">{{ "MAIN_PAGE" | translate }}</span>
-          <img class="logo" [src]="'pomagamyua.svg' | path: 'imagesPath'" [alt]="'MAIN_PAGE' | translate" />
-        </a>
+        <span class="desktop-only header-element">
+          <a href="/" class="navbar-brand">
+            <span class="visually-hidden">{{ "MAIN_PAGE" | translate }}</span>
+            <img
+              class="logo"
+              [src]="'kprm.svg' | path: 'imagesPath'"
+              [alt]="'CHANCELLERY_OF_PRIME_MINISTER' | translate"
+            />
+          </a>
+        </span>
+        <span class="desktop-only header-element">
+          <a href="/" class="navbar-brand">
+            <span class="visually-hidden">{{ "MAIN_PAGE" | translate }}</span>
+            <img class="logo" [src]="'pomagamyua.svg' | path: 'imagesPath'" [alt]="'MAIN_PAGE' | translate" />
+          </a>
+        </span>
+      </div>
+      <div class="d-flex flex-row">
+        <div class="navbar-item dropdown header-element">
+          <a
+            class="navbar-link dropdown-toggle"
+            href="#"
```

Comment:
For MVP2, we should look at all links with `href="#"` and either change them to real links or make them HTML buttons with custom CSS.